basic/knowbe4_training/knowbe4.tf

```diff
@@ -0,0 +1,40 @@
+# An AWS Secrets Manager Secret to hold your KnowBe4 API Key. Set the value with:
+# aws secretsmanager put-secret-value --secret-id "$sym/{local.environment_name}/knowbe4-api-key" --secret-string "YOUR-KNOWBE4-API-KEY"
+resource "aws_secretsmanager_secret" "knowbe4_api_key" {
+  name        = "sym/${local.environment_name}/knowbe4-api-key"
```

Comment:
Looks like the docs don't quite match this, they just say `"main/knowbe4-api-key"`. Is that intentional?

Reply:
The other API keys (PagerDuty, Tailscale, etc) started with `sym/...` so I assumed it was convention for the `examples` repo 🤷‍♀️

Reply:
Yes, I do think they should have `sym/...`; customers having Sym-specific secrets namespaced is generally good practice. Let's make the docs match?